integrationm/scripts/_wrkfl_work_queue.groovy

```diff
@@ -12,58 +14,58 @@ import java.util.concurrent.TimeUnit
 
 @Field static workQueuesCache = CacheBuilder.newBuilder()
         .expireAfterWrite(5, TimeUnit.MINUTES)
-        .build();
+        .build() as Cache<String, List<RecordmSearchHit>>
 
 if (msg.product == "recordm" && msg.type == "Work Queues") {
     log.info("Invalidating WorkQueues cache")
     workQueuesCache.invalidateAll()
 }
 
 
-if (msg.product == "recordm" && !DEFS_TO_IGNORE.contains(msg.type)) {
-
-    def query = "specific_data.raw:\"${msg.type}\" AND launch_condition:*"
-    def wqHits = getWorkQueueshits(query)
-
-    wqHits.each { hit ->
-        def launchCondition = hit.value('Launch Condition')
-        log.debug("Launch content: ${launchCondition.split("\n")}")
-
-        try {
-            if (evaluate(launchCondition)) {
-                def possibleStates = hit.value('Possible States')
-                log.info("Launch condition true: ${launchCondition.split("\n")}")
-                log.info("Possible States: ${possibleStates}")
-
-                Map updates = [:]
-                updates["Customer Data"] = msg.instance.id
-                // log.info("MMM>>>"+hit.value("Specific Data Main Customer Data Field"))
-                if (hit.value("Specific Data Main Customer Data Field")) {
-                    updates["Main Customer Data"] = "" + msg.value(hit.value("Specific Data Main Customer Data Field"))
-                }
-                updates["Work Queue"] = hit.value("id")
-                updates["State"] = possibleStates[0]
-
-                log.info("updates: ${updates}")
-                recordm.create("Work Item", updates)
-            }
-        } catch (Exception e) {
-            log.error("couldn't evaluate launchcondition {{" +
-                    "WQ: ${hit.value('id')}:${hit.value('Code')}:${hit.value('Name')}, " +
-                    "condition: ${launchCondition}" +
-                    "}}", e);
-        }
-    }
-}
+if (msg.product != "recordm" || DEFS_TO_IGNORE.contains(msg.type)) {
```

Comment:
Este if  está exactamente ao contrário do que deve ser !
 o If que estava era correcto

Reply:
It was a test that I forgot to rollback the change. Fixed!